snmp/datadog_checks/snmp/config.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, instance, warning, log, global_metrics, mibs_path, profiles,
         if profile:
             if profile not in profiles:
                 raise ConfigurationError("Unknown profile '{}'".format(profile))
-            self.metrics.extend(profiles[profile]['definition'])
+            self.metrics.extend(profiles[profile]['definition']['metrics'])
```

Comment:
Can you "detect" if user is using the old config and log an error message how user can fix the issue ?

Reply:
We could, but it should break cleanly enough, and we mostly hope it's not done.

Reply:
Ok, sounds good then.